e2e/volumegroupsnapshot.go

```diff
@@ -176,6 +176,14 @@ func (rvgs *rbdVolumeGroupSnapshot) ValidateResourcesForCreate(vgs *groupsnapapi
 
 func (rvgs *rbdVolumeGroupSnapshot) ValidateResourcesForDelete() error {
 	validateOmapCount(rvgs.framework, 0, rbdType, defaultRBDPool, volumesType)
+	validateOmapCount(rvgs.framework, 0, rbdType, defaultRBDPool, snapsType)
+	validateOmapCount(rvgs.framework, 0, rbdType, defaultRBDPool, groupSnapsType)
+	validateRBDImageCount(rvgs.framework, 0, defaultRBDPool)
+
+	err := waitToRemoveImagesFromTrash(rvgs.framework, defaultRBDPool, deployTimeout)
+	if err != nil {
+		framework.Failf("failed to validate rbd images in pool %s trash: %v", defaultRBDPool, err)
```

Comment:
return error message

Reply:
its already logged at the caller, lets remove the extra logging from here

Reply:
I am not sure if `framework.Failf()` really returns, does the test not get aborted when it is called?